geopandas/tests/test_geodataframe.py

```diff
@@ -504,9 +503,8 @@ def test_dataframe_to_geodataframe(self):
         gf2 = df.set_geometry('location', crs=self.df.crs, drop=True)
         assert isinstance(df, pd.DataFrame)
         assert isinstance(gf2, GeoDataFrame)
-        assert gf2.geometry.name == 'geometry'
-        assert 'geometry' in gf2
-        assert 'location' not in gf2
+        assert gf2.geometry.name == 'location'
+        assert 'location' in gf2
```

Comment:
I wasn't sure about this change.  This is a deviation from previous behavior.  It makes sense to me though.

Reply:
yes, that seems to make sense

Reply:
Might be good to clarify that in the docstring though